integrations/argo-workflows/documentation.yaml

```
@@ -0,0 +1,72 @@
+exporter_type: included
+app_name_short: Argo Workflows Controller
+app_name: {{app_name_short}}
+app_site_name: Argo Workflows Controller
+app_site_url: https://argoproj.github.io/argo-workflows/
+exporter_name: the Argo Workflows Controller exporter
+exporter_pkg_name: argo-workflows
+exporter_repo_url: https://github.com/argoproj/argo-workflows/
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. To verify that {{exporter_name}} is emitting 
+  metrics on the expected endpoints, do the following:
+
+  1. Set up port forwarding by using the following command:
+
+  <pre class="devsite-click-to-copy">
+  kubectl -n {{namespace_name}} port-forward {{pod_name}} 9090
+  </pre>
+
+  2. Access the endpoint `localhost:9090/metrics` by using the browser
+  or the `curl` utility in another terminal session.
+dashboard_available: true
+minimum_exporter_version: v3.4.3
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: argo-workflows-controller
+    labels:
+      app.kubernetes.io/name: argo-workflows-controller
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+  + - port: 9090
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+  +     app: workflow-controller
```

`podmonitoring_config` should not have `+` deltas.

```suggestion
  apiVersion: monitoring.googleapis.com/v1
  kind: PodMonitoring
  metadata:
    name: argo-workflows-controller
    labels:
      app.kubernetes.io/name: argo-workflows-controller
      app.kubernetes.io/part-of: google-cloud-managed-prometheus
  spec:
    endpoints:
    - port: 9090
      scheme: http
      interval: 30s
      path: /metrics
    selector:
      matchLabels:
        app: workflow-controller
```